TerminalBoard.cpp

```diff
@@ -132,3 +102,21 @@ TerminalBoard::showBoard()
 		std::cout << std::endl;
 	std::cout << "Score: " << fTarget->Score() << std::endl;
 }
+
+void
+TerminalBoard::KeyDown(const char *bytes, int32 numBytes)
+{
+	BMessenger messenger(NULL, fTarget);
+	if ( numBytes == 1 ) {
+		int32 c = bytes[0];
+		if (c == 'w' || c == 'a' || c == 's' || c == 'd'){
```

Comment:
You probably also want to check for `B_UP_ARROW` and friends so people can use the arrow keys, see `headers/os/interface/InterfaceDefs.h`.